packages/synapse-react-client/src/lib/utils/functions/getEndpoint.ts

```diff
@@ -53,7 +53,8 @@ export type SynapseStack = 'production' | 'staging' | 'development' | 'mock'
 
 export const MOCK_REPO_ORIGIN =
   'https://mock-repo.sagebase.org' satisfies string
-const MOCK_PORTAL_ORIGIN = 'https://mock-repo.sagebase.org/' satisfies string
+const MOCK_PORTAL_ORIGIN =
+  'https://portal-dev.dev.sagebase.org/' satisfies string
```

Comment:
This should be different than mock-repo but it probably shouldn't go to portal-dev. Maybe change to `mock-portal`